lib/react-popout.jsx

```diff
@@ -54,7 +53,7 @@ export default class PopoutWindow extends React.Component {
    * @param props
    */
   componentWillReceiveProps(props){
-    props.containerId && (CONTAINER_ID = props.containerId);
+    props.containerId && (this[_CONTAINER_ID] = props.containerId);
```

Comment:
the `_CONTAINER_ID` now being instance state should avoid any issues if you have more than one popout on the page with different `props.containerId`.

Nice one